releases.yaml

```
@@ -1,4 +1,177 @@
+            e2fsprogs: 1.46.2-2+wb1
+            e2fsprogs-l10n: 1.46.2-2+wb1
+            emmcparm: 5.0.1
+            firmware-realtek: 20170823-1~bpo9+1
+            flash-simcom-a76xx: 1.1.1
+            frpc: 0.52.3
+            frps: 0.52.3
+            fuse2fs: 1.46.2-2+wb1
+            gir1.2-modemmanager-1.0: 1.20.0-1~bpo11+1-wb108
+            gir1.2-nm-1.0: 1.42.4-1~bpo11+1-wb102
+            hubpower: '1.0'
+            intrahouse: 5.15.29
+            intrascada: 5.15.29
+            knxd: 0.14.51-1
+            knxd-dev: 0.14.51-1
+            knxd-examples: 0.11.15-1-wb1
+            knxd-tools: 0.14.51-1
+            libateccssl: '0.1'
+            libateccssl1.1: 0.2.5
+            libcom-err2: 1.46.2-2+wb1
+            libext2fs-dev: 1.46.2-2+wb1
+            libext2fs2: 1.46.2-2+wb1
+            libfdt1: 1.6.0-1
+            libmm-glib-dev: 1.20.0-1~bpo11+1-wb108
+            libmm-glib-doc: 1.20.0-1~bpo11+1-wb108
+            libmm-glib0: 1.20.0-1~bpo11+1-wb108
+            libmodbus-dev: 3.1.6-wb2
+            libmodbus5: 3.1.6-wb2
+            libnm-dev: 1.42.4-1~bpo11+1-wb102
+            libnm0: 1.42.4-1~bpo11+1-wb102
+            libsigrok4: 0.5.2-4
+            libsigrokdecode4: 0.5.3-4
+            libss2: 1.46.2-2+wb1
+            libwbmqtt1-4: 4.4.2
+            libwbmqtt1-4-dev: 4.4.2
+            libwbmqtt1-4-test-utils: 4.4.2
+            linux-libc-dev: 5.10.35-wb164
+            logsave: 1.46.2-2+wb1
+            mmc-utils: 0+git20180327.b4fe0c8c-1+wb1
+            mobile-broadband-provider-info: 20230416-1
+            modbus-utils: 1.2.10
+            modbus-utils-rpc: 1.2.2
+            modemmanager: 1.20.0-1~bpo11+1-wb108
+            modemmanager-dev: 1.20.0-1~bpo11+1-wb108
+            modemmanager-doc: 1.20.0-1~bpo11+1-wb108
+            mplc4-oni-plc-w: 1.3.3.15735
+            mqtt-tools: 1.4.4
+            network-manager: 1.42.4-1~bpo11+1-wb102
+            network-manager-config-connectivity-debian: 1.42.4-1~bpo11+1-wb102
+            network-manager-dev: 1.42.4-1~bpo11+1-wb102
+            nodejs: 16.18.1-deb-1nodesource1
+            python-gsmmodem-new: '1:0.11'
+            python-gspread: 1:0.4.1
+            python-mosquitto: 1.3.4-2contactless1
+            python-nrf24: 1.0+1
+            python-wb-io: 1.2.3
+            python3-aioblescan: 0.2.14
+            python3-intelhex: 2.3.0-1
+            python3-json-rpc: 1.9.2.wb1
+            python3-mosquitto: 1.3.4-2contactless1
+            python3-mqttrpc: 1.2.4
+            python3-paho-socket: 0.0.3-1
+            python3-umodbus: 1.0.4-1+wb1
+            python3-wb-common: 2.1.2
+            python3-wb-diag-collect: 1.8.6
+            python3-wb-mcu-fw-updater: 1.10.11
+            python3-wb-mqtt-metrics: 0.3.3
+            python3-wb-nm-helper: 1.33.1
+            python3-wb-test-suite-deps: 1.18.4
+            python3-wb-update-manager: 1.3.2
+            python3-zpl: 0.1.10-wb101
+            rapidscada: 6.1.4-1
```

```suggestion
            rapidscada: 6.2.1-1
```
в тестинге обновилось уже